spark-jobs/src/main/scala/za/co/absa/enceladus/standardization/StandardizationExecution.scala

```diff
@@ -222,9 +219,10 @@ trait StandardizationExecution extends CommonJobExecution {
 
   //scalastyle:off parameter.number
 
-  private def ensureSplittable(df: DataFrame, path: String, schema: StructType)
-                              (implicit spark: SparkSession, fsUtils: DistributedFsUtils): DataFrame = {
-    if (fsUtils.isNonSplittable(path)) {
+  private def ensureSplittable(df: DataFrame, input: PathWithFs, schema: StructType)
+                              (implicit spark: SparkSession): DataFrame = {
+    implicit val fsUtils = input.fileSystem.toFsUtils
+    if (fsUtils.isNonSplittable(input.path)) {
       convertToSplittable(df, schema)
```

Comment:
If you do not mind touching `StandardizationExecution.convertToSplittable`, I would also make `fsUtils` not implicit. Your new arch makes it obsolete and I favour your new approach.

Reply:
OK, done.